main.sh

```diff
@@ -122,7 +122,7 @@ print_config() {
 
 		Monitoring
 		  * Install Prometheus : $prom_install_msg
-		  * Install Grafana    : $GRAFANA_ENABLE
+		  * Install Grafana + Kepler dashboard   : $GRAFANA_ENABLE
```

Comment:
IMO, if `Install Grafana + Kepler dashboard`'s parameter as GRAFANA_ENABLE it looks strange.

Reply:
so are we saying we want a separate flag for the Kepler dashboard? or maybe I didn't understand the original intention in https://github.com/sustainable-computing-io/local-dev-cluster/pull/79#pullrequestreview-2032883413 
to change the output hint  :(